pages/[slug].js

```diff
@@ -56,6 +56,10 @@ function Page({ data: initialData = {}, preview, token }) {
   const pageData = data?.page || data?.[0];
   const slug = pageData?.slug;
 
+  useEffect(() => {
+    localStorage.setItem("preview", preview);
```

Comment:
Hi Ronix, 

I think we can make use of the `preview` prop value to check if the current page is in preview mode or not. 

You can try to verify if we're on preview mode through the `Open Preview` from the studio or add the `/api/preview?secret=${secret}&slug=${slug}` to the site URL. 
![Screen Shot 2022-11-24 at 12 09 55 PM](https://user-images.githubusercontent.com/78787873/203692416-15e7cbf7-73e0-4ffb-aea7-967aa9c0bb66.png)

To test the live URL (not in preview mode), you can click the live URL badge or access the URL directly from chrome incognito mode. 
![Screen Shot 2022-11-24 at 12 10 07 PM](https://user-images.githubusercontent.com/78787873/203692444-f6d9bacc-a55d-4d90-bef7-aa0f8c8f777f.png)

What do you think?